FoodFriends/app/src/main/res/layout/activity_main.xml

```diff
@@ -30,14 +30,11 @@
 
     <androidx.fragment.app.FragmentContainerView
         android:id="@+id/tvPlaceholder"
-        android:layout_width="419dp"
-        android:layout_height="611dp"
-        android:layout_marginTop="60dp"
+        android:layout_width="421dp"
```

Comment:
Not sure how to comment on the whole file. But generally we'd want the main activity screen as simple as possible and control everything through fragments. ie: You'd have a single fragment container spanning width and height of the screen. 

For functionality like logout: I'd suggest having another tab like settings. Or even have a button in your profiles fragment that invokes the logout functionality

Reply:
I have moved to log out functionality into the profile fragment and the activity main now has an empty fragment